corpus/convert.py

```diff
@@ -326,3 +330,69 @@ def run(self):
                     segment.fullname() in segments_whitelist
                 ):
                     f.write(segment.orth + "\n")
+
+
+class CorpusToRawWavHDFJob(Job):
+    """
+    Gets audio files from a Bliss corpus and stores them as HDF file
+    compatible with the returnn HDFDataset
```

Comment:
Well, the simple hdf writer claims this, I did not test compatibility with returnn yet.

Reply:
As the SimpleHDFWriter comes from RETURNN itself, it should hold the claim, if not this is a RETURNN bug.